xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/informationinline.vm

```diff
@@ -1,40 +1,55 @@
 ###
 ### Display document information
 ###
 <div id="informationcontent" class="xwikiintracontent">
-<div id="_information">
-  <div id="informationleft" class="half column">
-    <dl>
-      #if($tdoc.realLocale != $doc.realLocale)
-        <dt>$services.localization.render('core.viewers.information.translationCreation', [$tdoc.realLocale])</dt>
-        <dd>$services.localization.render('core.viewers.information.translationCreationData', [$xwiki.getUserName($tdoc.creator), $xwiki.formatDate($tdoc.creationDate)])</dd>
-      #end
-      <dt>$services.localization.render('core.viewers.information.includedPages')</dt>
-      <dd>
-        #set($pages = $tdoc.includedPages)
-        #if ($pages.isEmpty())
-          ## We display a "no included pages" message to avoid having a blank information panel when there is nothing
-          <span class="noitems">$services.localization.render('core.viewers.information.noIncludedPages')</span>
-        #else
-          #displayDocumentList($pages true $blacklistedSpaces)
+  <div id="_information">
+    <div id="informationleft" class="col-xs-12 col-sm-6">
+      <dl>
+        #if($tdoc.realLocale != $doc.realLocale)
+          <dt>$services.localization.render('core.viewers.information.translationCreation', [$tdoc.realLocale])</dt>
+          <dd>$services.localization.render('core.viewers.information.translationCreationData', [$xwiki.getUserName($tdoc.creator), $xwiki.formatDate($tdoc.creationDate)])</dd>
         #end
-      </dd>
-    </dl>
-  </div>## informationleft
-  <div id="informationright" class="half column">
-    <dl>
-      <dt>$services.localization.render('core.viewers.information.backlinks')</dt>
-      <dd>
-        #set($blinks = $sorttool.sort($doc.getBacklinks()))
-        #if ($blinks.isEmpty())
+        <dt>$services.localization.render('core.viewers.information.includedPages')</dt>
+        <dd>
+          #set($pages = $tdoc.includedPages)
+          #if ($pages.isEmpty())
+          ## We display a "no included pages" message to avoid having a blank information panel when there is nothing
+            <span class="noitems">$services.localization.render('core.viewers.information.noIncludedPages')</span>
+          #else
+            #displayDocumentList($pages true $blacklistedSpaces)
+          #end
+        </dd>
+        <dt>$services.localization.render('core.viewers.information.pageReference')</dt>
+        <dd id="information-reference">
+          <span class="noitems">$services.localization.render('core.viewers.information.pageReference.tips')</span>
+          <div class="reference-group">
+            <div class="btn btn-default" id="button-inout" title="$services.localization.render('core.viewers.information.pageReference.globalButton')">
+              $services.icon.renderHTML('world')
+            </div>
+            <span class="form-control" id="reference-value">$doc.getDocumentReference().getLocaleDocumentReference()</span>
```

Comment:
The reference needs to be XML encoded. Also, relying on ``toString()`` to get the string reference is a bit risky.

Reply:
> relying on `toString()` to get the string reference is a bit risky.

what do you suggest? using a string resolver?

Reply:
My suggestion is to serialize the reference. BTW, ``getLocaleDocumentReference`` [is deprecated](https://github.com/xwiki/xwiki-platform/blob/xwiki-platform-11.0/xwiki-platform-core/xwiki-platform-model/src/main/java/org/xwiki/model/reference/DocumentReference.java#L351). You should use ``getLocalDocumentReference`` (which makes more sense anyway).

```suggestion
            <span class="form-control" id="reference-value">$escapetool.xml($services.model.serialize($doc.getDocumentReference().getLocalDocumentReference()))</span>
```

Reply:
Hm indeed I meant serializer, not resolver. My bad. Thanks for the clarification :) Interesting to see how easy it is in velocity to use deprecated stuff...

Reply:
> Interesting to see how easy it is in velocity to use deprecated stuff

But you get a warning in the log when you do that.